setup.py

```diff
@@ -5,7 +5,7 @@
 Slycot wraps the SLICOT library which is used for control and systems analysis.
 
 """
-from __future__ import division, print_function
+from skbuild import setup
```

Comment:
The installation documentation will need to let users know of this new requirement, as well as cmake (and any other build tools that now need to be installed). I personally think that skbuild should be an optional requirement because a user can build it without this if they so desire.

Reply:
I added information on scikit-build and cmake requirements to the readme file.

I don't think it is feasible to maintain two buildsystems. I got to use cmake/skbuild after repeatedly trying to use and tweak the numpy distutils, however I found no way to specify flang to the distutils buildsystem, while flang works wonderfully for Windows builds. 

The buildsystem does not rely on conda; if skbuild and cmake are available, local builds work fine.

Reply:
Ok, seems reasonable.